pipelines/ci/create-github-status.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: create-github-status
```

Comment:
Namespace?

Reply:
As we discussed, it'll go into the "current namespace", the other tasks also go into the "current tasks", so it doesn't feel quite right to just fix this one, either we should fix 'em all, or ensure that the namespace is set (which it should be with `oc`).